tests/phpunit/tests/media.php

```diff
@@ -3798,6 +3798,83 @@ function() use ( &$result ) {
 
 		$this->assertFalse( $result );
 	}
+
+	/**
+	 * @ticket 55443
+	 */
+	public function test_wp_print_image_mime_fallback_script_if_content_has_updated_images() {
+
+		if ( ! wp_image_editor_supports( array( 'mime_type' => 'image/webp' ) ) ) {
+			$this->markTestSkipped( 'This test requires WebP support.' );
+		}
+
+		remove_all_actions( 'template_redirect' );
+		do_action( 'template_redirect' );
```

Comment:
Can we get an inline comment about why this is necessary?

Reply:
Removed now it not needed.